CHANGELOG.md

```diff
@@ -0,0 +1,176 @@
+# Quarkus extension for MicroProfile Language Server Changelog
+
+## 0.9.0 (September 21, 2020)
+
+Since `0.9.0`, quarkus-ls becomes an extension of [lsp4mp](https://github.com/eclipse/lsp4mp), the Language Server for Eclipse MicroProfile, see the upstream [lsp4mp CHANGELOG](https://github.com/eclipse/lsp4mp/CHANGELOG.md#010-september-21-2020)

```

Comment:
add missing period at the end of the sentence.

Reply:
fixed